lib/client/socket.js

```diff
@@ -38,15 +38,22 @@ module.exports = ({ url, token, filters, config, identifyingMetadata }) => {
   io.on('error', ({ type, description }) => {
     if (type === 'TransportError') {
       logger.error({ type, description }, 'Failed to connect to broker server');
+    } else {
+      logger.warn({ type, description }, 'Error on websocket connection');
     }
+
   });
   io.on('open', () => {
-    logger.info({ token, identifyingMetadata },
+    logger.info({ url, token, identifyingMetadata },
       'identifying with broker server');
     const clientData = { token, metadata: identifyingMetadata };
     io.send('identify', clientData);
   });
 
+  io.on('close', () => {
+    logger.warn({ url, token }, 'server connection closed');
```

Comment:
Why a `warn` and not an `info` for this one?

Reply:
From the broker client's perspective, losing the connection to the server is not an 'all good' event. On the other hand, it is also not the client's fault (as far as it knows for now).
Using `warn` log level here to make these logs stand out in case they happen.